src/web/modules/services/detail.njk

```diff
@@ -114,6 +116,14 @@
       href: "/stripe/basic/create?service=" + service.external_id
       })
     }}
+    <h2 class="govuk-heading-s">Test account</h2>
+    <p class="govuk-body">Create Stripe test account for service:</p>
```

Comment:
Does this description add any new information? Looks like all of that text is included on the button action. This can probably be removed

Reply:
👍🏽 removed